### PR TITLE
CNV-17405: installation requirements reorganization

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.6-4.11

Issue: [CNV-17405](https://issues.redhat.com/browse/CNV-17405)

Link to docs preview: [Preparing your cluster for OpenShift Virtualization](http://file.tlv.redhat.com/apinnick/CNV17405-installation-requirements-reorg/virt/install/preparing-cluster-for-virt.html)

Additional information: No QE required.

### CODE REVIEW

**Comment by apinnick** [comment]
@fabiand The writer who was originally assigned to this task left Red Hat so I gave it a go. Could you please review the PR? Thanks.

File: modules/virt-platform-node-storage-requirements.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+[id="virt-platform-node-storage-requirements_{context}"]
+= Platform, node, and storage requirements
+
+You can install {VirtProductName} on a cluster that complies with the following platform, node, and storage requirements:
+
+* Platforms:
+
+** On-premise bare metal.
+** Amazon Web Services bare metal instance. Bare metal instances offered by other cloud providers are not supported.
+
```
Comment (dominikholler) [inline modules/virt-platform-node-storage-requirements.adoc:15]: I like the cleanliness of the two lines above.

File: modules/virt-platform-node-storage-requirements.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+:FeatureName: Installing OpenShift Virtualization on an AWS bare metal instance
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+endif::[]
+
+* Red Hat Enterprise Linux CoreOS (RHCOS) installed on worker nodes. RHEL Linux 7 and 8 nodes are not supported.
+
+* CPUs:
+
+** Supported by link:https://catalog.redhat.com[RHEL 8]. RHEL 9 does not support certain CPUs.
```
Comment (dominikholler) [inline modules/virt-platform-node-storage-requirements.adoc:26]: Why is `RHEL 9 does not support certain CPUs.` relevant here?

File: modules/virt-platform-node-storage-requirements.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+** Support for Intel 64 or AMD64 CPU extensions.
+** Intel VT or AMD-V hardware virtualization extensions enabled.
+** NX (no execute) flag enabled.
+
+* CSI-enabled cloud storage.
```
Comment (dominikholler) [inline modules/virt-platform-node-storage-requirements.adoc:31]: I am not sure from a technical point of view. Probably the storage also has to be supported by OpenShift. In the best case, we would have a link here.
@aglitke What is your point of view?

File: modules/virt-platform-node-storage-requirements.adoc (hunk 2 above)
Comment (apinnick) [reply modules/virt-platform-node-storage-requirements.adoc:26]: Probably not relevant. I'll take it out.

File: modules/virt-platform-node-storage-requirements.adoc (hunk 3 above)
Comment (apinnick) [reply modules/virt-platform-node-storage-requirements.adoc:31]: I will change to OpenShift-supported storage. 

Should CSI be mentioned?

**Review by sjhala-ccs — COMMENTED** [review]
A couple of suggestions, otherwise LGTM

**Pull request merged**